examples/axpy_pipelined.py

```diff
@@ -64,7 +64,6 @@ def run_axpy_pipeline():
 
     with tf.Session() as sess:
         # Initialize the globals and the dataset
-        sess.run(tf.global_variables_initializer())
```

Comment:
This line is not needed any more?

Reply:
there are no variables in this example. not sure why it was there in the first place